core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -36,6 +36,9 @@ private CatalogProperties() {
   public static final String CLIENT_POOL_CACHE_EVICTION_INTERVAL_MS = "client.pool.cache.eviction-interval-ms";
   public static final long CLIENT_POOL_CACHE_EVICTION_INTERVAL_MS_DEFAULT = TimeUnit.MINUTES.toMillis(5);
 
+  public static final String APP_ID = "app-id";
+  public static final String USER = "user";
```

Comment:
Can you revert the changes to this file, please? This doesn't look like a necessary change.

Reply:
hmmm...strange, seems unsynced from master branch